src/test/suite/Utils.ts

```diff
@@ -28,11 +28,20 @@ export async function ensureExtensionActivated() {
 	if (extension !== null && extension !== undefined) {
 		await waitUntil(() => {
 			return extension.isActive;
-		}, 45000).catch( () => {
-			extension.activate().then( () => {
-				assert.ok("Camel K extension is ready to go");
-			});
+		}, 45000).catch(() => {
+			console.log('Extension has not started automatically, we will force call to activate it.');
 		});
+		if(!extension.isActive) {
+			await extension.activate();
+			await waitUntil(() => {
+				return extension.isActive;
+			}, 45000).catch(() => {
+				console.log('Extension is not active even after calling activate explicitely.');
```

Comment:
I am wondering if it wouldn't be easier to just fail with that message directly in the catch. It will fail in the next if isActive anyways, so why just do a console.log? What do you think?

Reply:
yeah, I kind of agree here. I suspect that we'll end up with a missed failure otherwise

Reply:
Only if the extension.isActive out of the sudden will be true. At least that's how I interpret that code passage.

Reply:
I read it as...

* explicitly activate the extension
* wait 45000ms to see if it's successful, if not, log a message
* then check it a final time to see if it's successful, if not, fail

I think the timeout should be sufficient. if it's not successful in that timeframe, just fail it

also -- `explicitely` is misspelled -- should be `explicitly`

Reply:
> I am wondering if it wouldn't be easier to just fail with that message directly in the catch. It will fail in the next if isActive anyways, so why just do a console.log? What do you think?

I tried and it was failing

Reply:
rethinking about it, I think I see why this piece of code is not clear (for you and for the me-one-day-after). I will rework that.